source/extensions/http/header_validators/envoy_default/http1_header_validator.cc

```diff
@@ -99,6 +91,20 @@ Http1HeaderValidator::validateResponseHeaderEntry(const HeaderString& key,
   return validateGenericHeaderValue(value);
 }
 
+ValidationResult Http1HeaderValidator::validateContentLengthAndTransferEncoding(
+    const ::Envoy::Http::RequestOrResponseHeaderMap& header_map) {
+  if (header_map.TransferEncoding()) {
+    if (header_map.ContentLength() &&
```

Comment:
nit: Your call but consider using a single `if` statement here with all the clauses.

Reply:
done